### ISSUE TITLE
Crash when using intersection types

### ISSUE THREAD

**Issue by AvivRubys** [brigand/babel-plugin-flow-react-proptypes#40]
The transformation crashes whenever intersection types are used (which are used in react native, making this transform impossible to use with react native).

Reproducing example:
**index.js**
```
type Props = {
    someProp: {} & {}
};
class MyComponent extends React.Component {
    props: Props;
}
```
**.babelrc**
```
{
  "presets": ["react"],
  "plugins": ["flow-react-proptypes", "transform-flow-strip-types"]
}
```
Result:
```
Error: index.js: babel-plugin-flow-react-proptypes: Encountered an unknown node in the type definition. Node: {"type":"IntersectionTypeAnnotation","start":29,"end":36,"loc":{"start":{"line":2,"column":14},"end":{"line":2,"column":21}},"types":[{"type":"ObjectTypeAnnotation","start":29,"end":31,"loc":{"start":{"line":2,"column":14},"end":{"line":2,"column":16}},"callProperties":[],"properties":[],"indexers":[]},{"type":"ObjectTypeAnnotation","start":34,"end":36,"loc":{"start":{"line":2,"column":19},"end":{"line":2,"column":21}},"callProperties":[],"properties":[],"indexers":[]}]}
    at convertToPropTypes (C:\Projects\test\node_modules\babel-plugin-flow-react-proptypes\lib\convertToPropTypes.js:109:11)
    at C:\Projects\test\node_modules\babel-plugin-flow-react-proptypes\lib\convertToPropTypes.js:20:15
    at Array.map (native)
    at convertToPropTypes (C:\Projects\test\node_modules\babel-plugin-flow-react-proptypes\lib\convertToPropTypes.js:15:31)
    at convertNodeToPropTypes (C:\Projects\test\node_modules\babel-plugin-flow-react-proptypes\lib\index.js:27:43)
    at PluginPass.TypeAlias (C:\Projects\test\node_modules\babel-plugin-flow-react-proptypes\lib\index.js:159:25)
    at newFn (C:\Projects\test\node_modules\babel-traverse\lib\visitors.js:276:21)
    at NodePath._call (C:\Projects\test\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\Projects\test\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\Projects\test\node_modules\babel-traverse\lib\path\context.js:105:12)
```

**Comment by brigand** [comment]
@AvivRubys thanks for the report. I'm not quite sure what this should look like in the output. I might just mark it as PropTypes.any for now.

**Comment by brigand** [comment]
Published as 0.10.1.
